modules/installation-extracting-infraid.adoc

```diff
@@ -30,5 +42,36 @@ openshift-vw9j6 <2>
 installation files in.
 <2> The output of this command is your cluster name and a random string.
 +
-You need the output of this command to configure the provided CloudFormation
-templates and can use it in other AWS tags.
+You must use the output of this command to configure the provided {cp-template}
+templates and can use it in other {cp} tags.
+endif::[]
+
+ifeval::["{context}" == "installing-gcp-user-infra"]
+. To extract and view the infrastructure name from the Ignition config file
+metadata, run the following command:
++
+----
+$ jq -r .infraID /<installation_directory>/metadata.json <1>
+openshift-vw9j6 <2>
+----
+<1> For `<installation_directory>`, specify the path to the directory that you stored the
+installation files in.
+<2> The output of this command is your cluster name and a random string.
+
+. Export the following variables to be used by the provided {cp-template}
+templates in later steps:
++
+----
+$ export BASE_DOMAIN='<base_domain>'
+$ export BASE_DOMAIN_ZONE_NAME='<base_domain_zone_name>'
+$ export NETWORK_CIDR='10.0.0.0/16'
+$ export MASTER_SUBNET_CIDR='10.0.0.0/19'
+$ export WORKER_SUBNET_CIDR='10.0.32.0/19'
```

Comment:
With these additions, `modules/installation-extracting-infraid.adoc` and the `Extracting the infrastructure name` subject are no longer such a good match.  Maybe broaden the scope to... `Basic installation configuration` or some such?  Or punt the variables other than `INFRA_ID` off to a different location.

Reply:
Created a generic `modules/installating-user-infra-exporting-common-variables.adoc` and couched them both in a new "Exporting common variables" section:

http://file.rdu.redhat.com/~adellape/100419/gcp_upi/installing/installing_gcp_user_infra/installing-gcp-user-infra.html#installation-gcp-user-infra-exporting-common-variables